src/components/Services/DtsList.tsx

```diff
@@ -15,6 +15,8 @@ import { useAuth } from "react-oidc-context";
 import { Log } from "oidc-client-ts";
 import Link from "next/link";
 import Pagination from "../Pagination/Pagination";
+import { GenericEntityResourceApi, GenericStateEntityTypeIdNewStatePutRequest } from '../../openapi-client/apis/GenericEntityResourceApi';
+import TimedToast from '../TimedToast/TimedToast';
```

Comment:
IMHO I would suggest this component as well as another one that is for managing pagination. You will place them in a site with several reusable components so that they can be used more comfortably.